src/main/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersController.java

```diff
@@ -48,11 +68,98 @@ public CasePaymentOrdersController(CasePaymentOrdersServiceImpl casePaymentOrder
+            code = 200,
+            message = ""
+        ),
+        @ApiResponse(
+            code = 400,
+            message = ValidationError.IDS_EMPTY,
+            examples = @Example(value = {
+                @ExampleProperty(
+                    value = "{\n"
+                        + "   \"status\": \"BAD_REQUEST\",\n"
+                        + "   \"errors\": [\n"
+                        + "   \"message\": \"" + ValidationError.IDS_EMPTY + "\",\n"
+                        + "   ]\n"
+                        + "}",
+                    mediaType = APPLICATION_JSON_VALUE
+                )
+            })
+        ),
+        @ApiResponse(
+            code = 401,
+            message = AuthError.AUTHENTICATION_TOKEN_INVALID
+        ),
+        @ApiResponse(
+            code = 403,
+            message = AuthError.UNAUTHORISED_S2S_SERVICE
+        )
+    })
+    public void deleteCasePaymentOrdersById(@RequestParam("ids") @Valid
+                                                  @NotEmpty(message = ValidationError.IDS_EMPTY)
+                                                          List<UUID> ids) throws CasePaymentIdentifierException {
+        casePaymentOrdersService.deleteCasePaymentOrdersByIds(ids);
+    }
+
+    @DeleteMapping(path = CASE_PAYMENT_ORDERS, params = "case-ids")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Delete case payment orders by case-ids", notes = "Delete case payment orders by case-ids")
+    @ApiResponses({
+        @ApiResponse(
+                code = 200,
+                message = ""
+        ),
+        @ApiResponse(
+            code = 400,
+            message = "One or more of the following reasons:"
+                + "\n1) " + ValidationError.CASE_IDS_INVALID
+                + "\n2) " + ValidationError.CASE_ID_INVALID_LENGTH
+                + "\n3) " + ValidationError.CASE_IDS_EMPTY,
+            examples = @Example(value = {
+                @ExampleProperty(
+                    value = "{\n"
+                            + "   \"status\": \"BAD_REQUEST\",\n"
+                            + "   \"errors\": [\n"
+                            + "      \"case-ids can not be empty,\"\n"
+                            + "      \"caseId invalid,\"\n"
+                            + "      \"invalid length\"\n"
+                            + "   ]\n"
+                            + "}",
+                    mediaType = APPLICATION_JSON_VALUE
+                )
+            })
+        ),
+        @ApiResponse(
+            code = 401,
+            message = AuthError.AUTHENTICATION_TOKEN_INVALID
+        ),
+        @ApiResponse(
+            code = 403,
+            message = AuthError.UNAUTHORISED_S2S_SERVICE
+        )
+    })
+    public void deleteCasePaymentOrdersByCaseId(@RequestParam("case-ids") @Valid
+                                                  @NotEmpty(message = ValidationError.CASE_IDS_EMPTY)
+                                                  List<
```

Comment:
@LuhnCheck(message = ValidationError.CASE_IDS_INVALID,
                                                        ignoreNonDigitCharacters = false)
                                                    @Size(min = 16, max = 16,
                                                            message = ValidationError.CASE_ID_INVALID_LENGTH)

that could be reduced to @ValidCaseId

Reply:
I did try this but it looks like `@ValidCaseId` only validates that the case id is a 16 digit string ? It doesn't validate that is is a valid Luhn number ?

Reply:
Resolving as code no longer applicable after recent refactored of `DELETE` to mirror `GET` endpoint validation functionality